zconf-ng.h.in

```diff
@@ -48,30 +55,29 @@
 /* Type declarations */
 
 
-#if defined(_WIN32)
-   /* If building or using zlib as a DLL, define ZLIB_DLL.
-    * This is not mandatory, but it offers a little performance increase.
-    */
-#  if defined(ZLIB_DLL) && (!defined(__BORLANDC__) || (__BORLANDC__ >= 0x500))
```

Comment:
I removed the `__BORLANDC__` checks since Borland C++ was released in 1996. I can add it back if need be.

Reply:
I would be better if removing borlandc was a separate commit that also removed all borlandc references.

Reply:
There's a separate PR for that https://github.com/zlib-ng/zlib-ng/pull/634

I'll rebase this one after it lands.